### DISCUSSION TITLE
Is it possible to search for template repositories, specifically?

### DISCUSSION THREAD

**Discussion by ferdnyc** [community/community#68572] [Repositories] [answered]
### Select Topic Area

Question

### Body

One underused feature of GitHub repositories is the ability to [create template repositories](https://docs.github.com/en/repositories/creating-and-managing-repositories/creating-a-template-repository) that contain skeleton code for projects in various languages/environments. Other users can then [create a repository from that template](https://docs.github.com/en/repositories/creating-and-managing-repositories/creating-a-repository-from-a-template) to clone it into a new repo without creating a fork relationship between the two.

This is great, but as I said it's somewhat underused. One main reason: How does one **_find_** template repositories to use? I can't seem to find any options in search (even the Advanced search) to search specifically for template repositories, or to filter search results to display only template repositories.

Similarly, the [create a new repository interface](https://github.com/new) has a "Repository template" selector at the top, but it only shows **your own** template repositories, not other users'.

Something like the [Actions Marketplace](https://github.com/marketplace), but for template repositories, would make those repositories a lot more discoverable, and the feature a lot more useful to the community.

Unless I'm just missing something?

**Comment by seyLu** [comment, marked as answer]
#### Hi @ferdny,

There's currently no way to filter search results in github to return only template repositories.

One workaround you can do is to search by topic, eg:
```bash
topic:template-repository
```
![image](https://github.com/community/community/assets/98249191/1f6e0d1b-be8a-4cc7-9de9-d7e427245465)

or searching terms in repo description, eg:
```bash
"template repository" in:description 
```
![image](https://github.com/community/community/assets/98249191/4b8ad695-8c8d-416d-9e01-5c6787239688)
